pkg/util/certificates/certificates.go

```diff
@@ -1,13 +1,136 @@
 package certificates
 
 import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
 	"crypto/x509"
+	"crypto/x509/pkix"
 	"encoding/pem"
+	"errors"
+	"math/big"
 	"time"
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/logd"
 )
 
+const (
+	intSerialNumberLimit  = 128
+	defaultCertExpiration = 7 * 24 * time.Hour
```

Comment:
why 7 days? cc @StefanHauth do we want short live Certificate here?

Reply:
I asked Stefan, and he mentioned not having an expiration date. I'll update it.